lib/spdy-transport/protocol/spdy/parser.js

```diff
@@ -172,7 +173,7 @@ Parser.prototype.onSynHeadFrame = function onSynHeadFrame(type,
 
     if (!isPush) {
       callback(null, {
-        type: 'HEADERS',
+        type: 'SYN_STREAM',
```

Comment:
I'd go with just HEADERS for both SYN_STREAM and SYN_REPLY. HTTP2 is using just HEADERS for both of them, and for continuation headers too. There is no complexity in handling this in `connection.js`.

TL;DR - let's leave it as it is ;)